SU-HSDO/suhumsci
# SHS-5583: empty headings in banners and spotlights

# READY FOR REVIEW

## Summary
Fix empty headings in banners and spotlights

## Need Review By (Date)
04/19

## Urgency
medium

## Steps to Test

1. Go to this pages

- https://music.suhumsci.loc/events/faculty-ensembles/st-lawrence-string-quartet
- https://music.suhumsci.loc/events/symposium-performing-intermedia
- https://music.suhumsci.loc/pianoseminar2023

2. Confirm that if there's no title, there's no an empty heading level being displayed in the markup
3. Edit each component and a title to confirm it's being displayed as expected


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
